Lib/pyclbr.py

```diff
@@ -202,11 +202,13 @@ def _create_tree(fullmodule, path, fname, source, tree, inpackage):
                 lineno, thisindent = start
                 # Close previous nested classes and defs.
                 while stack and stack[-1][1] >= thisindent:
+                    setattr(stack[-1][0], 'end_lineno', start[0] - 1)
```

Comment:
Why did you switch to using `setattr` here? This should be fine, right?

```suggestion
                    stack[-1][0].end_lineno = start[0] - 1
```

Also on the two other lines, below...

Reply:
I have to set the attribute, right? Otherwise, it results into the error: <> has no attribute end_lineno.

Reply:
>Also, just so you don't get discouraged: it's great that you're taking the time to contribute to CPython, but this module has no current experts or maintainers. I am not able to merge this PR once it's ready, and there is a fair chance that no active core developer would feel comfortable adding a new feature like this - especially without significant demand.

I received some positive responses on Stackoverflow for this. Don't you think there should be a starting and an end line for this?

Even if you think and abide by to go with the popular demand, should I opt for a survey, etc? Is there some platform where I can ask for votes: "Should there be end_lineno?"

Thank you

Reply:
> I have to set the attribute, right? Otherwise, it results into the error: <> has no attribute end_lineno.

They should both be equivalent. You're certain that applying my suggested change results in an error?

Reply:
Yes. My bad, apologies. I tried the direct approach to `stack[-1]` instead of `stack[-1][0]`.